app/buyers/views/buyers.py

```diff
@@ -362,6 +363,11 @@ def publish_brief(framework_slug, lot_slug, brief_id):
             question_and_answers['slug'] = section['id']
 
     unanswered_required, unanswered_optional = count_unanswered_questions(sections)
+
+    if sections.get_section('set-how-long-your-requirements-will-be-live-for') and \
+            sections.get_section('set-how-long-your-requirements-will-be-live-for').questions[0].answer_required:
+            unanswered_required -= 1
+
     if request.method == 'POST':
         if unanswered_required > 0:
```

Comment:
@idavidmcdonald The template won't render the publish button if they haven't answered the requirements length question so in the normal flow they wouldn't be able to get to the post route. There may be security implications if someone were to use postman or edit the html on the page to render the form. I could add another conditional to line 386 to check if it's a get request?